src/ReactiveUI/Activation/CanActivateViewFetcher.cs

```diff
@@ -33,7 +33,7 @@ public class CanActivateViewFetcher : IActivationForViewFetcher
         /// <returns>An observable tracking whether the view is active.</returns>
         public IObservable<bool> GetActivationForView(IActivatableView view)
         {
-            if (!(view is ICanActivate canActivate))
+            if (view is not ICanActivate canActivate)
```

Comment:
Probably worth adding a test for this condition to make sure it can handle non-IActivatableView's